config/locales/en.yml

```diff
@@ -4240,6 +4244,8 @@ en:
       builtin: Built-in instance application
       confidential: Confidential
       singular: "OAuth application"
+      scope: "Scopes:"
```

Comment:
Not an error introduced in this PR, but I noticed that it seemed inconsistent to see only scopes with a trailing `:` here. And indeed it's also inconsistent in the UI:

![image](https://github.com/user-attachments/assets/6cba19e7-2481-4110-9e3d-0f13ab4ff3ef)

I'd suggest to drop the colon here, so that it doesn't slip into all translations before being fixed.